data-scripts/create-dummy-data.mjs

```diff
@@ -0,0 +1,235 @@
+import fetch from 'node-fetch';
```

Comment:
this file should not exist anymore - it was renamed to data-scripts/create-dummy-data.js (mjs->js)

pull down my version of data-scripts/create-dummy-data.js and make sure your repo has the same thing then you can delete the .mjs file